misc/make-deps.sh

```diff
@@ -102,10 +102,11 @@ go get github.com/blynn/nex				# for lexing
 go get golang.org/x/tools/cmd/goyacc			# formerly `go tool yacc`
 go get golang.org/x/tools/cmd/stringer			# for automatic stringer-ing
 go get golang.org/x/lint/golint				# for `golint`-ing
+go get golang.org/x/tools/cmd/goimports		# for fmt
```

Comment:
This is needed on OSX? (It works on Linux already)

Reply:
This if needed for Linux as well. At least on a fresh Ubuntu install I have it was.

Reply:
`gofmt` is installed as part as a regular installation for me:

```
$ sudo dnf provides `which gofmt`
gcc-go-8.2.1-6.fc28.x86_64 : Go support
Repo        : updates
Matched from:
Filename    : /usr/bin/gofmt

golang-bin-1.10.7-1.fc28.x86_64 : Golang core compiler tools
Repo        : updates
Matched from:
Filename    : /usr/bin/gofmt

gcc-go-8.0.1-0.20.fc28.x86_64 : Go support
Repo        : fedora
Matched from:
Filename    : /usr/bin/gofmt

golang-bin-1.10.1-1.fc28.x86_64 : Golang core compiler tools
Repo        : fedora
Matched from:
Filename    : /usr/bin/gofmt

```

Is ubuntu missing this? Are you using `gofmt` ?

Reply:
```
~/g/s/g/p/mgmt (macos|✔) $ dpkg -L golang-1.10-go|grep -E 'goimports|gofmt'
/usr/lib/go-1.10/bin/gofmt
```
`gofmt` is but `goimports` isn't and that is what is used in `test/test-gofmt.sh`